python-phoenixdb/phoenixdb/types.py

```diff
@@ -298,6 +298,8 @@ def from_column(column):
 
     @staticmethod
     def _from_jdbc(jdbc_code):
+        if jdbc_code == 0:
```

Comment:
This is a bit ugly.
Would it be possible to add this to the type maps instead ?

Reply:
addressed